src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -1,39 +1,43 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-abstract class Planta(val anioObtencionSemilla: Int, var altura: Float) {
-  fun esFuerte() = this.horasDeSolQueTolera() > 10
+abstract class Planta(val anioObtencionSemilla: Int, val altura: Float) {
+    fun esFuerte() = this.horasDeSolQueTolera() > 10
 
-  fun parcelaTieneComplicaciones(parcela: Parcela) =
-    parcela.plantas.any { it.horasDeSolQueTolera() < parcela.horasSolPorDia }
-
-  abstract fun horasDeSolQueTolera(): Int
-  abstract fun daSemillas(): Boolean
+    abstract fun horasDeSolQueTolera(): Int
+    abstract fun daSemillas(): Boolean
 }
 
 class Menta(anioObtencionSemilla: Int, altura: Float) : Planta(anioObtencionSemilla, altura) {
-  override fun horasDeSolQueTolera() = 6
-  override fun daSemillas() = this.esFuerte() || altura > 0.4
+    override fun horasDeSolQueTolera() = 6
+    override fun daSemillas() = altura > 0.4
 }
 
-class Soja(anioObtencionSemilla: Int, altura: Float, val esTransgenica: Boolean) : Planta(anioObtencionSemilla, altura) {
-  override fun horasDeSolQueTolera(): Int  {
-    // ¡Magia de Kotlin! El `when` es como un `if` pero más poderoso:
-    // evalúa cada línea en orden y devuelve lo que está después de la flecha.
-    val horasBase = when {
-      altura < 0.5  -> 6
-      altura < 1    -> 7
-      else          -> 9
+open class Soja(anioObtencionSemilla: Int, altura: Float) : Planta(anioObtencionSemilla, altura) {
+    override fun horasDeSolQueTolera(): Int {
+        // ¡Magia de Kotlin! El `when` es como un `if` pero más poderoso:
+        // evalúa cada línea en orden y devuelve lo que está después de la flecha.
+        val horasBase = when {
+            altura < 0.5 -> 6
+            altura < 1 -> 7
+            else -> 9
+        }
+
+        return horasBase
```

Comment:
Acá tampoco hace falta usar un `val`:
```suggestion
        return when {
            altura < 0.5 -> 6
            altura < 1 -> 7
            else -> 9
        }
```